### PR TITLE
Lt 884 add application documents

### PR DESCRIPTION
**Pull request by 5pence**
review, has design signoff

### CODE REVIEW

File: templates/cases/case/views/other-documents.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                        <span class="govuk-visually-hidden">{% get_string 'cases.manage.documents.download_document' %}</span>
+                        {{ additional_document.name }}<br>
+                    </a>
+                    {% endif %}
+                </td>
+            </tr>
+        {% endfor %}
+		</tbody>
+	</table>
+
```
Comment (BenAndrew-Kainos) [inline templates/cases/case/views/other-documents.html]: Remove blank lines

**Comment by BenAndrew-Kainos** [comment]
Should we also write an e2e test for the application-case page as a whole?

File: ui_automation_tests/helpers/seed_data.py [hunk 2]
```diff
@@ -135,6 +135,12 @@ class SeedData:
             's3_key': env('TEST_S3_KEY'),
             'size': 0,
             'description': 'document for test setup'
+        },
+        "additional_documents": {
```
Comment (phlptt) [inline ui_automation_tests/helpers/seed_data.py]: singular

File: ui_automation_tests/helpers/seed_data.py [hunk 3]
```diff
@@ -261,6 +267,11 @@ def add_consignee_document(self, draft_id):
         self.make_request("POST", url='/drafts/' + draft_id + '/consignee/document/',
                           headers=self.export_headers, body=data)
 
+    def add_additional_documents(self, draft_id):
```
Comment (phlptt) [inline ui_automation_tests/helpers/seed_data.py]: singular document

File: ui_automation_tests/helpers/seed_data.py (hunk 2 above)
Comment (5pence) [reply ui_automation_tests/helpers/seed_data.py]: happy to change though you can have more than one additional document

File: ui_automation_tests/helpers/seed_data.py (hunk 3 above)
Comment (5pence) [reply ui_automation_tests/helpers/seed_data.py]: happy to change though you can have more than one additional document

Comment (5pence) [reply ui_automation_tests/helpers/seed_data.py]: done

File: ui_automation_tests/helpers/seed_data.py (hunk 2 above)
Comment (5pence) [reply ui_automation_tests/helpers/seed_data.py]: done

File: ui_automation_tests/fixtures/apply_for_application.py [hunk 4]
```diff
@@ -52,13 +52,16 @@ def apply_for_standard_application(driver, request, api_url, context):
...
+
     )
     api.submit_application()
     context.app_id = api.context['application_id']
     context.case_id = api.context['case_id']
+    context.end_user = api.context['end_user']
     context.consignee = api.context['consignee']
     context.third_party = api.context['third_party']
     context.ultimate_end_user = api.context['ultimate_end_user']
+
```
Comment (BenAndrew-Kainos) [inline ui_automation_tests/fixtures/apply_for_application.py]: test comment

**Pull request merged**